src/repairnator-pipeline/src/test/java/fr/inria/spirals/repairnator/process/step/repair/TestNopolRepair.java

```diff
@@ -58,7 +59,8 @@ public void tearDown() throws IOException {
         FileHelper.deleteFile(tmpDir);
     }
 
-    @Test
+    @Ignore
+    //@Test
```

Comment:
avoid comments + @Ignore is enough. could you add a comment to explain why it is ignored?

Reply:
Only `@Ignore` didn't work for some reason. I commented `@Test` just to make it skip. I'll debug that.